From d2846fdcce9b8de0edecdf0e06a4b86fc8de542c Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Wed, 6 Apr 2016 10:24:58 +0200
Subject: [PATCH] plymouth/plymouth-pretrigger.sh: also trigger acpi subsystem

Trigger the acpi subsystem. This will ensure hv_vmbus gets loaded before
plymouth is started, which will make the graphics device become
available before plymouth is started too (and the keyboard ! which might
also be important for plymouth in some setups).

https://bugzilla.redhat.com/show_bug.cgi?id=1218130
---
 modules.d/50plymouth/plymouth-pretrigger.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules.d/50plymouth/plymouth-pretrigger.sh b/modules.d/50plymouth/plymouth-pretrigger.sh
index 4a3fcdf..161f424 100755
--- a/modules.d/50plymouth/plymouth-pretrigger.sh
+++ b/modules.d/50plymouth/plymouth-pretrigger.sh
@@ -5,7 +5,7 @@ if ! getarg rd_NO_PLYMOUTH; then
     # first trigger graphics subsystem
     udevadm trigger --attr-match=class=0x030000 >/dev/null 2>&1
     # first trigger graphics and tty subsystem
-    udevadm trigger --subsystem-match=graphics --subsystem-match=drm --subsystem-match=tty >/dev/null 2>&1
+    udevadm trigger --subsystem-match=graphics --subsystem-match=drm --subsystem-match=tty --subsystem-match=acpi >/dev/null 2>&1
 
     udevadm settle --timeout=30 >/dev/null 2>&1
     [ -c /dev/zero ] || mknod -m 0666 /dev/zero c 1 5
